openhtf/plugs/usb/__init__.py

```diff
@@ -36,9 +38,27 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('remote_usb', 'Ethersyncy or other',default_value=None)
+
+_LOG = logging.getLogger(__name__)
 
+class GeneralUsbAttributeError(Exception):
+  """Indicates the usb class/subclass/protocol info was invalid."""
+
+def _get_usb_serial(port_addr):
+  """Get a usb serial based on the Cambrionix unit/port address in configuration."""
+  cmd = 'esuit64 -t "DEVICE INFO,%s"' % port_addr.strip()
+  info = commands.getstatusoutput(cmd)[1]
```

Comment:
I believe the 'commands' module is deprecated and we should probably use 'subprocess' instead. Especially if we intend on supporting Python3 eventually.